WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -974,6 +997,19 @@ - (void)loginValidationFailedWithError:(NSError *)error
     }
 }
 
+- (NSString *)getTagsCountPresentableString:(NSInteger)tagCount
+{
+    NSString *format;    
+    if (tagCount == 1) {
+        format = NSLocalizedString(@"%@ Tag", @"The number of tags in the writting settings. Singular. %@ is a placeholder for the number");
+    } else {
+        format = NSLocalizedString(@"%@ Tags", @"The number of tags in the writting settings. Plural. %@ is a placeholder for the number");
+    }
```

Comment:
I know I'm being nitpicky, but we could avoid the else clause if one of the strings was used to initialize `format`

Reply:
Done! :)

Reply:
💥 !